code/divide_conquer/src/factorial/factorial.cpp

```diff
@@ -0,0 +1,11 @@
+#include<iostream> 
+using namespace std; 
+  
+// function to find factorial of given number 
+
+unsigned int factorial(unsigned int n) 
+{ 
+    if (n == 0) 
```

Comment:
I think you should add brackets, an if statement only includes code that is on the same line or enclosed in brackets in the following lines